aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -551,11 +562,6 @@ void gemm<at::BFloat16>(CUDABLAS_GEMM_ARGTYPES(at::BFloat16)) {
   float fbeta = beta;
   _cublasAdjustLdLevel3(transa, transb, m, n, k, &lda, &ldb, &ldc);
   GEMM_CHECK_ARGVALUES(at::BFloat16);
-  cublasMath_t cublas_flags = CUBLAS_DEFAULT_MATH;
-  if (!at::globalContext().allowBF16ReductionCuBLAS()) {
-    cublas_flags = static_cast<cublasMath_t>(cublas_flags | CUBLAS_MATH_DISALLOW_REDUCED_PRECISION_REDUCTION);
-  }
-  TORCH_CUDABLAS_CHECK(cublasSetMathMode(handle, cublas_flags));
```

Comment:
@zstreet87 Not clear why we needed to remove this snippet if it is being conditionalized only for CUDA above.